src/index.html

```diff
@@ -5,8 +5,363 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div>
+        <a href="#">
+          <img
+            src="\images\logo.png"
+            alt="logo"
+            class="header__logo"
+          >
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">apple</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link" data-qa="nav-hover">ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">гаджеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">планшеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">фото</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div class="card" data-qa="card">
```

Comment:
better name for this block - `catalog`

And you could delete container div in line 63, and use instead of it - mix. More detail in video.
And add changes to styles according to changes in html.

```suggestion
    <main class="catalog">
        <div class="card catalog__card" data-qa="card">
```

[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)